### ISSUE TITLE
Program breaks while using `gitmoji -c`

### ISSUE THREAD

**Issue by yogeshvar** [carloscuesta/gitmoji-cli#324]
Hello @carloscuesta!

Issue:
Exit 1 while using `gitmoji -c`
Please find the attachment given below.
```Error: Total of weights cannot exceed 1
    at e.value (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/fuse.js/dist/fuse.js:9:3409)
    at new e (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/fuse.js/dist/fuse.js:9:2634)
    at filterGitmojis (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/lib/utils/filterGitmojis.js:24:16)
    at Object.source (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/lib/commands/commit/prompts.js:27:56)
    at AutocompletePrompt.search (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt/index.js:177:32)
    at AutocompletePrompt._run (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt/index.js:65:10)
    at /Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/prompts/base.js:57:12
    at new Promise (<anonymous>)
    at AutocompletePrompt.run (/Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/prompts/base.js:56:12)
    at /Users/mags/.nvm/versions/node/v12.13.1/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/ui/prompt.js:98:27
```

**OS**:
**gitmoji-cli -v**: 3.2.1
**Node version**: v12.13.1

**Error Message**:

![image](https://user-images.githubusercontent.com/17572818/76204217-f9a2ef80-621d-11ea-968f-b9bf875b2b03.png)

**Comment by D0C702WH0** [comment]
Same behavior here

Gitmoji Version: 3.2.1
Node Version : v13.8.0

**Comment by gustavopch** [comment]
Same behavior.

Gitmoji: both v3.2.0 and v3.2.1.
Node: both v10.19.0 and v12.16.1.

```
Error: Total of weights cannot exceed 1
    at e.value (/usr/local/lib/node_modules/gitmoji-cli/node_modules/fuse.js/dist/fuse.js:9:3409)
    at new e (/usr/local/lib/node_modules/gitmoji-cli/node_modules/fuse.js/dist/fuse.js:9:2634)
    at filterGitmojis (/usr/local/lib/node_modules/gitmoji-cli/lib/utils/filterGitmojis.js:24:16)
    at Object.source (/usr/local/lib/node_modules/gitmoji-cli/lib/commands/commit/prompts.js:27:56)
    at AutocompletePrompt.search (/usr/local/lib/node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt/index.js:177:32)
    at AutocompletePrompt._run (/usr/local/lib/node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt/index.js:65:10)
    at /usr/local/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/prompts/base.js:57:12
    at new Promise (<anonymous>)
    at AutocompletePrompt.run (/usr/local/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/prompts/base.js:56:12)
    at /usr/local/lib/node_modules/gitmoji-cli/node_modules/inquirer/lib/ui/prompt.js:98:27
```

Probably not related, but I started to have this issue after replacing https://github.com/nvm-sh/nvm with http://npm.im/n.

Gitmoji v3.1.0 works fine tho.

**Comment by carloscuesta** [comment]
This must be related by #303 @adriencaccia. I'll try to take a look at this

**Comment by yupswing** [comment]
I'm experiencing the same behavior: 

```sh
❯ gitmoji -v
3.2.1
❯ node -v
v13.9.0
```
(note: I'm using `nodenv`)

Error with `-c`, `-i` or `-s`; `-l` is fine, probably because there is no `search`.

```sh
❯ gitmoji -s bug
query bug
Error: Total of weights cannot exceed 1
    at e.value (/home/yupswing/.nodenv/versions/13.9.0/lib/node_modules/gitmoji-cli/node_modules/fuse.js/dist/fuse.js:9:3409)
```

**Comment by xItsDanger** [comment]
Same happens to me, ill try to find something in the commit @carloscuesta mentioned

**Comment by xItsDanger** [comment]
> Same happens to me, ill try to find something in the commit @carloscuesta mentioned

I tried gitmoji 3.2.0, gives the same error. 3.1.0 is fine, so the error has to be from PR #267
Idk if im right, im new to this

**Comment by xItsDanger** [comment]
I think i almost got it, when the error prompts in cmd, we can see this 2 lines:
```js
    at filterGitmojis (/home/andres/.nvm/versions/node/v12.16.1/lib/node_modules/gitmoji-cli/lib/utils/filterGitmojis.js:24:16)
    at Object.source (/home/andres/.nvm/versions/node/v12.16.1/lib/node_modules/gitmoji-cli/lib/commands/commit/prompts.js:27:56)
```

So the error might be coming from filterGitmojis.js, or prompts.js.
I think it can be the filterGitmojis file, beacause it was added when 3.2.0 released
But dont trust me, im novice at this.

**Comment by carloscuesta** [comment]
Fixed at 3.2.2

Thanks 🎉
